__tests__/app.test.js

```diff
@@ -248,4 +248,63 @@ describe("All requests", () => {
                 });
         });
     });
+    describe("PATCH /api/articles/:article_id", () => {
+        it("200 should be returned when article_id and body validation are valid", () => {
+            return request(app)
+                .patch("/api/articles/1")
+                .expect(200)
+                .send({ inc_votes: 2 })
+                .then(({ body }) => {
+                    const { article } = body;
+                    expect(article["votes"]).toBe(102);
+
+                    expect(article).toHaveProperty("article_id", expect.any(Number));
+                    expect(article).toHaveProperty("title", expect.any(String));
+                    expect(article).toHaveProperty("topic", expect.any(String));
+                    expect(article).toHaveProperty("author", expect.any(String));
+                    expect(article).toHaveProperty("body", expect.any(String));
+                    expect(article).toHaveProperty("created_at", expect.any(String));
+                    expect(article).toHaveProperty("votes", expect.any(Number));
+                    expect(article).toHaveProperty("article_img_url", expect.any(String));
+                });
+        });
+
+        test("400 should be returned when article_id is NaN.", () => {
```

Comment:
maybe your description should be reflect that this could work for any id that isn't an id